styles/components.css

```diff
@@ -9,3 +15,448 @@ p {
+  grid-gap: var(--spacer-lg);
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  justify-items: center;
+}
+
+/* einheitliche Höhe bei unterschiedlich viel Text? min-height? */
+
+.landingpage-teaser__card {
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  padding: var(--spacer-lg) var(--spacer-base) var(--spacer-lg) var(--spacer-base);
+  color: var(--brand-white);
+  text-align: start;
+}
+
+.landingpage-teaser__card--purple {
+  background-color: var(--brand-purple1);
+}
+
+.landingpage-teaser__card--blue {
+  background-color: var(--brand-blue1);
+}
+
+.landingpage-teaser__card--white {
+  background-color: var(--brand-white);
+}
+
+.landingpage-card__title {
+  font-size: var(--font-size-base);
+  font-weight: var(--font-weight-bold);
+  line-height: var(--heading-line-height);
+}
+
+.landingpage-card__title--black {
+  margin-top: var(--spacer-md);
+  color: var(--brand-black);
+}
+
+.landingpage-card__twitterhandle {
+  margin-top: var(--spacer-xxs);
+  color: var(--brand-black);
+  font-size: var(--font-size-sm);
+  font-weight: var(--font-weight-regular);
+  line-height: var(--paragraph-line-height);
+}
+
+.landingpage-card__subtitle {
+  margin-top: var(--spacer-base);
+  font-size: var(--font-size-sm);
+  font-weight: var(--font-weight-regular);
+  line-height: var(--paragraph-line-height);
+}
+
+.landingpage-card__subtitle--black {
+  color: var(--brand-black);
+}
+
+.landingpage-teaser__card__avatar {
+  width: auto;
+  height: 5rem;
+  border-radius: 100%;
+  background-color: var(--brand-purple3);
+}
+
+/* Noch offen: Hover, visited */
+
+.landingpage-card__button {
+  display: inline-block;
+  height: auto;
+  padding: var(--spacer-xxs);
+  margin-top: var(--spacer-base);
+  color: var(--brand-white);
+  font-size: var(--font-size-sm);
+  font-weight: var(--font-weight-regular);
+  line-height: var(--paragraph-line-height);
+  text-decoration: none;
+}
+
+.landingpage-card__button--purple {
+  background-color: var(--brand-purple2);
+}
+
+.landingpage-card__button--blue {
+  background-color: var(--brand-blue2);
+}
+
+/* footer mobile */
+
+.footer {
+  display: flex;
+  justify-content: center;
+  background-color: var(--brand-black);
+}
+
+.footer__wrapper {
+  display: flex;
+  max-width: var(--content-width-xl);
+  flex-direction: column;
+  align-items: center;
+  padding: var(--spacer-lg);
+  color: var(--brand-white);
+}
+
+.footer__subtitle {
+  margin-top: var(--spacer-lg);
+  color: var(--brand-white);
+  font-size: var(--font-size-sm);
+  font-weight: var(--font-weight-regular);
+  line-height: var(--paragraph-line-height);
+  text-align: center;
+}
+
+.footer__logo {
+  width: var(--spacer-xxl);
+  height: auto;
+  align-content: center;
+  margin-top: var(--spacer-sm);
+}
+
+.footer__textlink {
+  align-items: center;
+  margin-top: var(--spacer-base);
+  color: var(--brand-white);
+  font-size: var(--font-size-sm);
+}
+
+/* breakpoint für größere Screens */
```

Comment:
Ich würde die Breakpoints jeweils zu ihrer Komponente schreiben. Sonst wird das zu unübersichtlich. eventuell überlegen wir auch mal, ob es Sinn macht, das File in mehrere Teile zu splitten. Es wird jetzt schon recht unübersichtlich.

Reply:
Nein, ein file für Header und da mobile und desktop rein.
Analog auch für die Cards. :)